install/helm/gloo/generate/values.go

```diff
@@ -84,6 +84,10 @@ type PodSpec struct {
 type JobSpec struct {
 	*PodSpec
 	ActiveDeadlineSeconds   *int              `json:"activeDeadlineSeconds,omitempty" desc:"Deadline in seconds for Kubernetes jobs."`
+	BackoffLimit            *int              `json:"backoffLimit,omitempty" desc:"Specifies the number of retries before marking this job failed. In kubernetes, defaults to 6"`
+	Completions             *int              `json:"completions,omitempty" desc:"Specifies the desired number of successfully finished pods the job should be run with."`
+	ManualSelector          *bool             `json:"manualSelector,omitempty" desc:"manualSelector controls generation of pod labels and pod selectors."`
```

Comment:
```suggestion
	ManualSelector          *bool             `json:"manualSelector,omitempty" desc:"Controls generation of pod labels and pod selectors."`
```
nit: for consistency with the other descriptions